src/View/Widgets/OverlayBar.vala

```diff
@@ -208,19 +208,19 @@ namespace Marlin.View {
                 if (folders_count > 1) {
                     str = _("%u folders").printf (folders_count);
                     if (files_count > 0) {
-                        str += ngettext (_(" and %u other item (%s) selected").printf (files_count, fsize),
-                                         _(" and %u other items (%s) selected").printf (files_count, fsize),
-                                         files_count);
+                        str += ngettext (" and %u other item (%s) selected",
+                                         " and %u other items (%s) selected",
+                                         files_count).printf (files_count, fsize);
                     } else {
                         str += _(" selected");
                     }
```

Comment:
Honestly I think we may want to avoid this way of constructing a translatable string but would be better to address in another PR.